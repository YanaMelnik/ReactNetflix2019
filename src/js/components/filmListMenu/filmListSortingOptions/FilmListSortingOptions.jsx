import React from 'react';

const FilmListSortingOptions = () => <form>
        <span>Sort by</span>
        <span>
            <input name='sort__by' type='radio' value='date'/>
            release date
        </span>
        <span>
            <input name='sort__by' type='radio' value='rating' checked/>
            rating
        </span>
    </form>;
export default FilmListSortingOptions;
