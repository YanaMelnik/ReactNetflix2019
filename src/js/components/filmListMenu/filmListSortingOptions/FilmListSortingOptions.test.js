import React from 'react';
import { shallow } from 'enzyme';
import FilmListSortingOptions from './FilmListSortingOptions';

describe('FilmListSortingOptions', () => {
    it('should render component correctly', () => {
        const component = shallow(<FilmListSortingOptions/>);

        expect(component).toMatchSnapshot();
    });
});
