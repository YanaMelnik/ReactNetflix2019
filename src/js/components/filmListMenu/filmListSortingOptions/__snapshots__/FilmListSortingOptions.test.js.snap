// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmListSortingOptions should render component correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
